<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monthlyPaid</class>
 <widget class="QDialog" name="monthlyPaid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblmonthlPaidTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Register new monthly paid employee&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblmonthlPaidName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Name&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(FirstName LastName)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtmonthlyPaidName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtmonthlyPaidSocialSecuritynumber">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblmonthlPaidsocialSecuritynumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Social security &lt;br/&gt;number&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(DDMMYY-123A)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtmonthlyPaidSalary">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblmonthlPaidsocialsalary">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Monthly &lt;br/&gt;Compensation&lt;/span&gt;&lt;br/&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;(Salary)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnaddMonthlPaid">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btncancelMonthlPaid">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
